organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml

```diff
@@ -123,42 +119,66 @@ Resources:
-                  - 'ec2:DescribeInstances'
-                  - 'ec2messages:GetEndpoint'
-                  - 'ec2messages:AcknowledgeMessage'
-                  - 'ec2messages:GetMessages'
-                  - 'ec2messages:FailMessage'
-                  - 'ec2messages:SendReply'
-                  - 'cloudformation:*'
-                  - 'resource-groups:ListGroupResources'
-                  - 'license-manager:ListLicenseConfigurations'
-                  - 'license-manager:UpdateLicenseSpecificationsForResource'
-                Resource: '*'
+                  - "organizations:ListAccountsForParent"
+                  - "organizations:ListOrganizationalUnitsForParent"
+                  - "organizations:ListChildren"
+                  - "tag:GetResources"
+                  - "ec2:DescribeInstances"
+                  - "ec2messages:GetEndpoint"
+                  - "ec2messages:AcknowledgeMessage"
+                  - "ec2messages:GetMessages"
+                  - "ec2messages:FailMessage"
+                  - "ec2messages:SendReply"
+                  - "cloudformation:*"
+                  - "resource-groups:ListGroupResources"
+                  - "license-manager:ListLicenseConfigurations"
+                  - "license-manager:UpdateLicenseSpecificationsForResource"
+                Resource: "*"
               - Effect: Allow
                 Action:
-                  - 'sts:AssumeRole'
-                Resource: !Sub 'arn:${AWS::Partition}:iam::*:role/OracleDbLTS-SystemsManagerAutomationExecutionRole'
+                  - "sts:AssumeRole"
+                Resource: !Sub "arn:${AWS::Partition}:iam::*:role/OracleDbLTS-SystemsManagerAutomationExecutionRole"
               - Effect: Allow
                 Action:
-                  - 's3:ListBucket'
-                  - 's3:GetObject'
-                  - 's3:DeleteObject'
-                  - 's3:PutObject'
-                  - 's3:PutObjectAcl'
-                Resource: 
+                  - "s3:ListBucket"
+                  - "s3:GetObject"
+                  - "s3:DeleteObject"
+                  - "s3:PutObject"
+                  - "s3:PutObjectAcl"
+                Resource:
                   - !GetAtt ArtifactsS3.Arn
                   - !Sub "arn:aws:s3:::${ArtifactsS3.Arn}/*"
               - Effect: Allow
                 Action:
-                  - 'iam:PassRole'
-                Resource: 
-                  - !Sub 'arn:${AWS::Partition}:iam::${AWS::AccountId}:role/OracleDbLTS-SystemsManagerAutomationAdministrationRole'
-                  - !Sub 'arn:${AWS::Partition}:iam::*:role/OracleDbLTS-SystemsManagerAutomationExecutionRole'
-  
+                  - "iam:PassRole"
+                Resource:
+                  - !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/OracleDbLTS-SystemsManagerAutomationAdministrationRole"
+                  - !Sub "arn:${AWS::Partition}:iam::*:role/OracleDbLTS-SystemsManagerAutomationExecutionRole"
+  StackSetAdministrationRole:
```

Comment:
Added This back into the template as it was not present in the yaml file that i got from aws

Reply:
that lines 157 to 180

Reply:
looks good